static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -562,10 +563,21 @@ const spanOperationRelativeBreakdownRenderer = (
   }
 
   let otherPercentage = 1;
-
+  let orderedSpanOpsBreakdownFields;
+  if (
+    eventView?.sorts?.[0].field &&
```

Comment:
Nit: pull this out into a variable so you don't have to retype the entire expression multiple times.

Reply:
Updated!